Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -160,7 +160,7 @@ const Contest = () => {
         [ participantsStateText, participantsValue ],
     );
 
-    useEffect(
+    useLayoutEffect(
```

Comment:
According to the documentation of react, this is not a proper way of using useLayoutEffect.